### PR TITLE
Changelog for patch release: 1.4.1

### PR DESCRIPTION
**Pull request by Zsailer**

### CODE REVIEW

**Review by blink1073 — APPROVED** [review]
Thanks!

**Comment by kevin-bates** [comment]
Thank you for addressing this so quickly!

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,24 @@
 
 All notable changes to this project will be documented in this file.
 
+## [1.5.0](https://github.com/jupyter-server/jupyter_server/tree/1.5.0) (2021-02-22)
```
Comment (vidartf) [inline CHANGELOG.md:5]: The title of the issue says 1.4.1, but here it says 1.5.0 ?

Comment (Zsailer) [reply CHANGELOG.md:5]: Oops, good catch! Fixing now

**Pull request merged**